packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -388,7 +388,7 @@ export default Mixin.create(EmberArray, MutableEnumerable, {
     @public
   */
   addObject(obj) {
-    if (!this.contains(obj)) {
+    if (!this.includes(obj)) {
```

Comment:
This is a small breaking change. I've left a note that it be called out in the deprecation guide https://github.com/emberjs/website/pull/2600#issuecomment-222124560. Here and `without`.